keyleak/disclose.py

```
@@ -0,0 +1,213 @@
+            return False
+        # Best-effort verification.
+        import tempfile
+
+        with tempfile.NamedTemporaryFile("wb", delete=False, suffix=".payload") as payload_file:
+            payload_file.write(payload)
+            payload_path = payload_file.name
+        with tempfile.NamedTemporaryFile("w", delete=False, suffix=".sig") as sig_file:
+            sig_file.write(sig)
+            sig_path = sig_file.name
+        try:
+            result = subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)